src/AppInstallerCommonCore/Public/winget/Filesystem.h

```diff
@@ -42,4 +42,7 @@ namespace AppInstaller::Filesystem
 
     // Gets the path of a known folder.
     std::filesystem::path GetKnownFolderPath(const KNOWNFOLDERID& id);
+
+    // Verifies that the target path is on the same volume as the current path.
+    bool IsSameVolume(const std::filesystem::path& current, const std::filesystem::path& target);

```

Comment:
nit: maybe the concept of "current" and "target" does not apply to IsSameVolume, path1 and path2?

Reply:
changed to 'path1' and 'path2'